aclk/agent_cloud_link.h

```diff
@@ -5,6 +5,7 @@
 
 #include "../daemon/common.h"
 #include "mqtt.h"
+#include "aclk_lws_wss_client.h"
```

Comment:
You can't do this - this file is included everywhere outside of the feature flag.

Reply:
Removed